compliance/cis_eks/rules/cis_3_2_6/rule.rego

```diff
@@ -0,0 +1,40 @@
+package compliance.cis_eks.rules.cis_3_2_6
+
+import data.compliance.cis_eks
+import data.compliance.lib.common
+import data.compliance.lib.data_adapter
+
+# Ensure that the --protect-kernel-defaults argument is set to true (Automated)
+# todo: If the --protect-kernel-defaults argument is not present, check that there is a Kubelet config file specified by --config, and that the file sets protectKernelDefaults to true.
```

Comment:
same here. I know it's from my rules but still

Reply:
Done